projects/ngrx.io/content/guide/store/configuration/runtime-checks.md

```diff
@@ -2,12 +2,13 @@
 
 Runtime checks are here to guide developers to follow the NgRx and Redux core concepts and best practices. They are here to shorten the feedback loop of easy-to-make mistakes when you're starting to use NgRx, or even a well-seasoned developer might make. During development, when a rule is violated, an error is thrown notifying you what and where something went wrong.
 
-`@ngrx/store` ships with four (4) built-in runtime checks:
+`@ngrx/store` ships with five (5) built-in runtime checks:
 
 - [`strictStateImmutability`](#strictstateimmutability): verifies that the state isn't mutated
 - [`strictActionImmutability`](#strictactionimmutability): verifies that actions aren't mutated
 - [`strictStateSerializability`](#strictstateserializability): verifies if the state is serializable
 - [`strictActionSerializability`](#strictactionserializability): verifies if the actions are serializable
+- [`strictActionWithinNgZone`](#strictActionWithinNgZone): verifies if actions are dispatched within NgZone
 
 These checks are all opt-in and will automatically be disabled in production builds.
```

Comment:
This line is not entirely accurate anymore seeing that the immutability checks are on by default. Will have a go at updating this.